### PR TITLE
Make new API changes to Email module

### PR DESCRIPTION
**Pull request by Maninda**
## Purpose
Make new API changes to Email module.

### CODE REVIEW

File: email-ballerina/commons.bal [hunk 1]
```diff
@@ -19,27 +19,27 @@ import ballerina/mime;
...
-# + replyTo - Reply To addresses
 # + attachments - Email attachements
-public type Email record {|
+public type Message record {|
     string[] to;
-    string[] cc?;
-    string[] bcc?;
     string subject;
+    string 'from;
     string|xml|json body;
```
Comment (shafreenAnfar) [inline email-ballerina/commons.bal:36]: string body right ?

File: email-ballerina/imap_listener_endpoint.bal [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+# + username - Email server access username
+# + password - Email server access password
+# + protocolConfig - IMAP4 protocol configuration
+# + pollingInterval - Periodic time interval to check new update
+# + cronExpression - Cron expression to check new update
+public type ImapListenerConfig record {|
+    string host;
+    string username;
+    string password;
+    ImapConfig? protocolConfig = ();
```
Comment (shafreenAnfar) [inline email-ballerina/imap_listener_endpoint.bal:161]: Don't have to have this inside this anymore right ?

File: email-ballerina/imap_listener_endpoint.bal [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    int pollingInterval = 60000;
```
Comment (shafreenAnfar) [inline email-ballerina/imap_listener_endpoint.bal:162]: `pollingIntervalInMillis` ?

File: email-ballerina/commons.bal (hunk 1 above)
Comment (Maninda) [reply email-ballerina/commons.bal]: Corrected.

File: email-ballerina/imap_listener_endpoint.bal (hunk 2 above)
Comment (Maninda) [reply email-ballerina/imap_listener_endpoint.bal]: Replaced with its inner fields.

File: email-ballerina/imap_listener_endpoint.bal (hunk 3 above)
Comment (Maninda) [reply email-ballerina/imap_listener_endpoint.bal]: Corrected.

**Review by niveathika — APPROVED** [review]

**Pull request merged**